website_rma/controllers/main.py

```diff
@@ -29,6 +30,9 @@ def _get_website_rma_product_domain(self, q):
         return [
             ('name', '=ilike', "%{}%".format(q or '')),
             ("sale_ok", "=", True),
+            '|',
+            ('bom_ids.type', '!=', 'phantom'),
```

Comment:
This is not valid, as you don't depend on `mrp` for doing this. You need to find another way.